regcore/layer.py

```diff
@@ -0,0 +1,22 @@
+class LayerParams(object):
```

Comment:
Good point on both counts -- this class originally had a lot more methods, but has been parred down over revisions.